packages/dx-react-grid-bootstrap3/src/templates/table-no-data-cell.jsx

```diff
@@ -1,62 +1,42 @@
 import * as React from 'react';
 import * as PropTypes from 'prop-types';
-import { getStickyPosition } from '../utils/css-fallback-properties';
 
-export class TableNoDataCell extends React.PureComponent {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      stickyPosition: getStickyPosition(),
-    };
-  }
-
-  componentDidMount() {
-    this.setState({ stickyPosition: getStickyPosition() });
-  }
-
-  render() {
-    const {
-      style,
-      colSpan,
-      getMessage,
-      tableRow,
-      tableColumn,
-      ...restProps
-    } = this.props;
-    const { stickyPosition } = this.state;
-
-    return (
-      <td
+export const TableNoDataCell = ({
+  style,
+  colSpan,
+  getMessage,
+  tableRow,
+  tableColumn,
+  ...restProps
+}) => (
+  <td
+    style={{
+      padding: '50px 0',
+      ...style,
+    }}
+    colSpan={colSpan}
+    {...restProps}
+  >
+    <div
+      style={{
+        display: 'inline-block',
+        position: 'fixed',
```

Comment:
'fixed' position is a bad idea because it's relative to a page not a grid

Reply:
When "No data" message is relative to grid, we get wrong result (see GIF). If we should'n make some relate to page, we need to relate on grid's container.
Also is message relative to page critical? I tried to resize, zoom and move window and didn't break this.
![GIF123123123123123123123](https://user-images.githubusercontent.com/43685423/64966771-13373000-d8a8-11e9-9f2b-372c10424217.gif)

Reply:
Well, try to set grid's width to 300px and a page height to 3000px

Reply:
I get it, I'll try another.